docs/cis_self_assessment.md

```diff
@@ -565,28 +562,17 @@ The apiserver, by default, does not authenticate itself to the kubelet's HTTPS e
 
 **Result:** Pass
 
-**Remediation:**
-Follow the Kubernetes documentation and set up the TLS connection between the
-apiserver and kubelets. Then, edit API server pod specification file
-`/etc/kubernetes/manifests/kube-apiserver.yaml` on the master node and set the
-kubelet client certificate and key parameters as below.
-
-``` bash
---kubelet-client-certificate=<path/to/client-certificate-file>
---kubelet-client-key=<path/to/client-key-file>
-```
-
 **Audit:**
+Run the below command on the master node.
 
-```
+```bash
 /bin/ps -ef | grep kube-apiserver | grep -v grep
 ```
 
-**Expected result**:
+Verify the output contains these 2 arguments.
```

Comment:
What you have is fine, but upsream cis doc has
```
Verify that the --kubelet-client-certificate and --kubelet-client-key arguments exist and they are set as appropriate.
```
I want to stay as aligned to the doc as is reasonable.